src/locales/en-US/list.ftl

```diff
@@ -200,11 +200,18 @@ ensure-logins-checked = Ensure the “Logins” checkbox is selected in sync pre
 setting-to-allow-sync = In order to allow your logins to be synced to other devices, this setting must be checked. <openprefs>Open sync preferences</openprefs></p>
 
 banner-promote-device =
-  .title = { -product-full-name }
-  .details = Download our mobile app to take your passwords with you everywhere.
-  .actionLabel = Learn More
+  .title = Take your passwords everywhere
+  .details = download our app for iOS or Android:
+
+# TODO is there an official localization we need to use for the alt text in other locales?
```

Comment:
For what it's worth, on mozilla.org we put these out for translations as normal strings.

Reply:
Great! In that case, I'll delete the TODO and let our localizers do their thing